backend/services/file_service.py

```
@@ -0,0 +1,31 @@
+        shutil.copyfileobj(upload_file.file, buffer)
+    return file_path
+
+
+def generate_secure_filename(user_id: int, file_ext: str) -> str:
+    return f"{user_id}_{secrets.token_hex(8)}{file_ext}"
+
+def remove_old_file_if_exists(file_path: str):
+    if os.path.exists(file_path):
+        os.remove(file_path)
+
+import re
```

Ruff (0.8.2)

Module level import not at top of file

(E402)